{{ cookiecutter.folder_name }}/.github/workflows/standard.yml

```diff
@@ -0,0 +1,29 @@
+name: PCDS Standard Testing
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+  pull_request:
+    branches:
+      - master
+  release:
+    types:
+      - created
+
+jobs:
+  standard:
+    uses: klauer/pcds-ci-helpers/.github/workflows/python-standard.yml@enh_gha
```

Comment:
TODO: change URL

Reply:
```suggestion
    uses: pcdshub/pcds-ci-helpers/.github/workflows/python-standard.yml
```

Reply:
Closed in [adfc066](https://github.com/pcdshub/cookiecutter-pcds-python/pull/37/commits/adfc06659ee243670ff9ecd49fd7218482d6d6f9)